std_msgs/Header header

float64 latitude	# latitude [deg]
float64 longitude	# longitude [deg]
float64 altitude	# altitude [m]
float32 heading 	# heading w.r.t. North [deg]
byte orientation_mode 	# orientation mode {TO_TARGET, HEADING_START, HEADING_DURING}
